includes/wikia/api/tests/ArticlesApiControllerTest.php

```diff
@@ -166,6 +166,49 @@ public function reorderForLinksDataProvider() {
 		];
 	}
 
+	public function replaceSectionIndexDataProvider() {
+		return [
+			// section text outside of heading tag
+			[
+				'<h2 section="1">Foo</h2><p>section=\'2\'</p>',
+				'3',
```

Comment:
These may better be number as the function documentation states that integer is expected as `$section`;